app/web_framework/cluster_master_application.py

```diff
@@ -151,6 +151,16 @@ def get(self, build_id):
         self.write(response)
 
 
+class _V2SubjobsHandler(_SubjobsHandler):
+    def get(self, build_id):
+        offset, limit = self.get_pagination_params()
+        build = self._cluster_master.get_build(int(build_id))
+        response = {
+            'subjobs': [subjob.api_representation() for subjob in build.all_subjobs(offset, limit)]
```

Comment:
Maybe we should consider a different name other than `all_subjobs`? Maybe like `get_subjobs` since it doesn't necessarily return _all_ the subjobs anymore